cli/replicaset/rebootstrap.go

```diff
@@ -0,0 +1,109 @@
+package replicaset
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/running"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// RebootstrapCtx rebootstrap operations arguments.
+type RebootstrapCtx struct {
+	// AppName application name the instance belongs to.
+	AppName string
+	// InstanceName is an instance name to re-bootstrap.
+	InstanceName string
+	// Confirmed is true if re-bootstrap confirmation is not required.
+	Confirmed bool
+}
+
+// cleanDataFiles removes snap, xlog and vinyl artifacts.
+func cleanDataFiles(instCtx running.InstanceCtx) error {
+	filesToRemove := []string{}
+	for _, pattern := range [...]string{
+		filepath.Join(instCtx.MemtxDir, "*.snap"),
+		filepath.Join(instCtx.WalDir, "*.xlog"),
+		filepath.Join(instCtx.VinylDir, "*.vinyl"),
+	} {
```

Comment:
From my experience vinyl creates `*.vylog` files. Why don't we delete them here also?